### ISSUE TITLE
Search text remains when switching context

### ISSUE THREAD

**Issue by drew-hoffman** [digimezzo/dopamine-windows#338]
If you view your songs, and search for an item, say "Nickelback", when you click on the playing song to go to the "Now Playing" playlist, your playlist is also filtered by "Nickelback". This is intuitive when you're flipping between the artists/genres/etc... tabs, but not so when you click on the current song to see the now playing playlist.

**Comment by mchowdhury** [comment]
Yeah, this seems to be bug that I encounter a lot too. Seems like the search bar needs to be cleared on switching to the "Now Playing" screen.
